Index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE >
+<html lang="en">
+
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+
+  <title>Protfolio Mobile-setup</title>
+
+  <!-- Load Styles' file -->
+  <link href="styles.css" rel="stylesheet">
+
+  <!-- Load Fonts -->
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+  <link href='http://fonts.googleapis.com/css?family=Roboto' rel='stylesheet' type='text/css'>
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&display=swap" rel="stylesheet">
+</head>
+
+<body>
+  <header class="header_section">
+
+    <img src="Sources/Union.png" alt="Unicon vector" id="unicon">
+    <span id="name">Omar Farouq</span>
+
+  </header>
+  <main>
+
+    
+
+    <section class="headline_section">
+
+        <img src="Sources/Group.png" alt="illustrations' vector"
+         id="bkg_illustrations">
```

Comment:
I think it would be good to include this image as a background image instead of using the `img` tag, It's good to include images of aesthetic value in a design as a background image and since this image is of an aesthetic value kindly include it as a background image.

The `img` tag should be used for images that have a functional purpose in our designs e.g clickable buttons or images used as part of the content in an article.